.github/workflows/deploy.yml

```diff
@@ -14,9 +14,10 @@ jobs:
         with:
           java-version: '11'
           distribution: 'temurin'
-
+          
```

Comment:
omit trailing spaces

Reply:
Sure, I am being messy because this is a draft MR.